app/concerns/dry_runnable.rb

```diff
@@ -0,0 +1,62 @@
+module DryRunnable
+  def dry_run!
+    class << self
+      prepend Sandbox
```

Comment:
Although save/save! is trapped, we can never be safe enough.  I've added a line to mark it as [readonly!](http://apidock.com/rails/ActiveRecord/Base/readonly%3F).